www/source/partials/docs/_reference-template-data.html.md.erb

```diff
@@ -59,49 +59,49 @@ Information about the current service's service group
+| svc.leader | [svc_member](#svc_member) | The current leader of the service group, if any (`null` otherwise) |
+| svc.update_leader | [svc_member](#svc_member) | The current update_leader of the service group, if any (`null` otherwise) |
 
 ## bind
 
-Exposes information about the service groups this service is bound to. Each key is the name of a bind, while each value is one of the objects described below
+Exposes information about the service groups this service is bound to. Expressed by referencing a bind as `bind.key.property`. E.g. `bind.elasticsearch.members`
 
 | Property | Type | Description |
 | -------- | ---- | ----------- |
-| first | [svc_member](#svc_member) | The first member of this service group. If the group is running in a leader topology, this will also be the leader. |
-| leader | [svc_member](#svc_member) | The current leader of this service group, if running in a leader topology |
-| members | array | All active members (`alive` and `suspect`) of the service group, across the entire ring. As of 0.56.0, does _not_ include `departed` or `confirmed` members |
+| bind.key.first | [svc_member](#svc_member) | The first member of this service group. If the group is running in a leader topology, this will also be the leader. |
+| bind.key.leader | [svc_member](#svc_member) | The current leader of this service group, if running in a leader topology |
+| bind.key.members | array | All active members (`alive` and `suspect`) of the service group, across the entire ring. As of 0.56.0, does _not_ include `departed` or `confirmed` members |
 
 ## Reference Objects
 
-Some of the template expressions referenced above return objects of a specific shape; for example, the `svc.me` and `svc.first` expressions return "service member" objects, and the `pkg` property of a service member returns a "package identifier" object. These are defined below.
+Some of the template expressions referenced above return objects of a specific shape; for example, the `svc.me` and `svc.first` expressions return "service member" objects, and the `pkg` property of a service member returns a "package identifier" object.
 
 ### package_identifier
 
-A Habitat package identifier, split apart into its constituent components
+A Habitat package identifier, split apart into its constituent components. Expressed as `origin/name/version/release`. E.g. `core/redis/3.2.4/20170514150022`
 
 | Property | Type | Description |
 | -------- | ---- | ----------- |
-| origin | string | The origin of the Habitat package |
-| name | string | The name of the Habitat package |
+| origin | string | The origin of the Habitat package. |
+| name | string | The name of the Habitat package. |
 | version | string | The version of the Habitat package |
 | release | string | The release of the Habitat package |
 
 ### svc_member
 
-Represents a member of a service group
+Represents a member of a service group. Expressed by referencing a svc_member object. E.g. `svc.me.alive`
```

Comment:
Should the commands in this table be `svc.me.thing`? If that's the correct pattern, then let's apply it consistently.

Reply:
Unfortunately no, you'd have to get an arbitrary `svc_member` object. But now that you mention it, it occurs to me that we're also doing this elsewhere, so I'll use an arbitrary designation for it.

Reply:
@kagarmoe Alright, I've updated the commit. I don't think there's a good way really of expressing it, but maybe this will be better than nothing.